src/components/date-picker/_date-picker.scss

```diff
@@ -0,0 +1,280 @@
+@import '../../globals/scss/colors';
+@import '../../globals/scss/css--reset';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/import-once';
+@import 'flatpickr.scss';
```

Comment:
Not sure how to test this as this is the only place I'm importing `flatpickr.scss`?